### PR TITLE
Replace Circle CI in CONTRIBUTING.MD

### PR DESCRIPTION
**Pull request by siemensikkema**

### CODE REVIEW

File: .github/CONTRIBUTING.MD [hunk 1]
```diff
@@ -36,4 +36,4 @@ New features are little more complex. If you come up with an idea, you should fo
 - [@tanner0101](https://github.com/tanner0101)
 - [@mcdappdev](https://github.com/mcdappdev)
```
Comment (jdmcd) [inline .github/CONTRIBUTING.MD:37]: Can you update this to my new username? Jdmcd please! I would do it but I'm on mobile, thanks!

**Review by jdmcd — CHANGES_REQUESTED** [review]
Just one quick comment!

File: .github/CONTRIBUTING.MD (hunk 1 above)
Comment (siemensikkema) [inline .github/CONTRIBUTING.MD:37]:
```suggestion
- [@mcdappdev](https://github.com/jdmcd)
```

File: .github/CONTRIBUTING.MD [hunk 2]
```diff
@@ -36,4 +36,4 @@ New features are little more complex. If you come up with an idea, you should fo
...
 
-See the [Vapor maintainers doc](https://github.com/vapor/vapor/blob/master/Docs/maintainers.md) for more information. 
\ No newline at end of file
+See the [Vapor maintainers doc](https://github.com/vapor/vapor/blob/master/Docs/maintainers.md) for more information.
```
Comment (siemensikkema) [inline .github/CONTRIBUTING.MD:39]:
```suggestion
See the [Vapor maintainers doc](https://github.com/vapor/vapor/blob/master/.github/maintainers.md) for more information. 
```

**Review by MrLotU — APPROVED** [review]

**Review by jdmcd — APPROVED** [review]

**Pull request merged**